src/common/__tests__/log-search-store.test.ts

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
This stateful store can probably be replaced with a reactive solutions, such as `asyncComputed`. Superficial, for now.